src/main/java/com/microsoft/spring/data/gremlin/telemetry/TelemetryTracker.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.telemetry;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import org.springframework.lang.NonNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.microsoft.spring.data.gremlin.telemetry.TelemetryProperties.PROPERTY_INSTALLATION_ID;
+import static com.microsoft.spring.data.gremlin.telemetry.TelemetryProperties.PROPERTY_VERSION;
+
+public class TelemetryTracker {
+
+    private static final String PROJECT_VERSION = TelemetryTracker.class.getPackage().getImplementationVersion();
+
+    private static final String PROJECT_INFO = "spring-cloud-azure" + "/" + PROJECT_VERSION;
```

Comment:
Is the `spring-cloud-azure` project info here expected?

Reply:
sorry, forget to change that, will fix it.